DMX.Core.Api.Tests.Unit/Services/Foundations/LabCommands/LabCommandServiceTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Linq.Expressions;
+using System.Runtime.Serialization;
+using DMX.Core.Api.Brokers.Loggings;
+using DMX.Core.Api.Brokers.Storages;
+using DMX.Core.Api.Models.Foundations.LabCommands;
+using DMX.Core.Api.Services.Foundations.LabCommands;
+using Microsoft.Data.SqlClient;
+using Moq;
+using Tynamix.ObjectFiller;
+using Xeptions;
+
+namespace DMX.Core.Api.Tests.Unit.Services.Foundations.LabCommands
+{
+    public partial class LabCommandServiceTests
+    {
+        private readonly Mock<IStorageBroker> storageBrokerMock;
+        private readonly Mock<ILoggingBroker> loggingBrokerMock;
+        private readonly ILabCommandService labCommandService;
+
+        public LabCommandServiceTests()
+        {
+            this.storageBrokerMock = new Mock<IStorageBroker>();
+            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+
+            this.labCommandService = new LabCommandService(
+                storageBroker: this.storageBrokerMock.Object,
+                loggingBroker: this.loggingBrokerMock.Object);
+        }
+
+        public static LabCommand CreateRandomLabCommand() =>
+            CreateLabCommandFiller().Create();
+
+        private static Filler<LabCommand> CreateLabCommandFiller()
+        {
+            var filler = new Filler<LabCommand>();
+
+            filler.Setup()
+                .OnType<DateTimeOffset>()
+                    .Use(GetRandomDateTimeOffset());
+
+            return filler;
+        }

```

Comment:
Move fillers always to the very bottom of your root test file. They are the least exposed outside of this file.